test_package_toml.py

```diff
@@ -0,0 +1,10 @@
+from package import get_build_system_from_pyproject_toml
```

Comment:
Can we move this file from the root to keep it clean?

Also, could you please update README with some instructions on how to run these? Or just delete it :)

Reply:
Moved python unit tests to `tests/` folder, add instructions to README.

And also added Github Action to run these tests against all supported python versions.